### PR TITLE
Add Cloudflare auto-minify to FAQ

### PR DESCRIPTION
**Pull request by jolheiser**
Adds an FAQ entry for the "why is my indent/spacing incorrect" question.

### CODE REVIEW

File: docs/content/doc/help/faq.en-us.md [hunk 1]
```diff
@@ -403,3 +403,8 @@ gitea doctor recreate-table
 ```
 
 It is highly recommended to back-up your database before running these commands.
+
+
+## Why are tabs/indents wrong when viewing files
+
+If you are using Cloudflare, turn off the auto-minify option in the dashboard.
```
Comment (Gusted) [inline docs/content/doc/help/faq.en-us.md]: Might want to mention that this option is under Speed -> Optimization:
image

And then specifically the HTML part:
image

**Review by Gusted — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
According to https://community.cloudflare.com/t/disable-minification-with-header/15294/14 `Cache-Control: no-transform` should also do it, I'll push a PR, but can not test it myself.

**Comment by silverwind** [comment]
See https://github.com/go-gitea/gitea/pull/20432.

**Review by silverwind — CHANGES_REQUESTED** [review]
See PR

**Comment by wxiaoguang** [comment]
I also prefer to add the header. Related issue:
* https://github.com/go-gitea/gitea/issues/19309

**Comment by jolheiser** [comment]
As long as we're sure that's the correct way to do it I'm fine leaving this out. 

Honestly might not be bad to leave a FAQ for older versions, at least for a while.

**Pull request merged**